libs/ui/elements/src/lib/metadata-info/metadata-info.component.html

```diff
@@ -30,7 +30,13 @@
 </ng-container>
 
 <gn-ui-expandable-panel
-  *ngIf="metadata.lineage"
+  class="metadata-origin"
+  *ngIf="
+    metadata.lineage ||
+    metadata.recordUpdated ||
+    metadata.updateFrequency ||
+    metadata.status
+  "
   [title]="'record.metadata.origin' | translate"
 >
   <p class="mb-5 pt-4 whitespace-pre-line break-words" gnUiLinkify>
```

Comment:
I think you need to also hide that block if there's no lineage, otherwise it makes a lot of whitespace